e3sm_to_cmip/__main__.py

```diff
@@ -98,9 +95,19 @@ class CLIArguments:
 
 class E3SMtoCMIP:
     def __init__(self, args: Optional[List[str]] = None):
+        # logger assignment is moved into __init__ AFTER the call to _parse_args
+        # to prevent the default logfile directory being created whenever a call
+        # to "--help" or "--version" is invoked.  Doing so, however, makes the
+        # logger unavailable to the functions in this class unless made global.
+        global logger
```

Comment:
Instead of instantiating `global logger`, we should have a conditional that checks if `--help`/`--version` is called and not create the default logfile directory.

Reply:
@tomvothecoder Right.  Originally, the "setup_logger" functions were global to modules like mpas.py, and would get executed by main: import mpas.py, LONG before arg-parsing happens.  So, the log-directory gets created before arg-parsing occurs.